test_scripts/WidgetSupport/CreateWindow/010_Success_create_primary_widget_with_a_different_windowName_value.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0216-widget-support.md
+--
+-- Description: Check that SDL ignores value of windowName for primary widget
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) "CreateWindow" is allowed by policies
+-- 3) App is registered and activated
+-- Step:
+-- 1) App creates a primary widget with type: "WIDGET" and different value of windowName via new RPC "CreateWindow"
+-- SDL does:
+--  - send UI.CreateWindow(params) request to HMI
+-- 2) HMI sends valid UI.CreateWindow response to SDL
+-- SDL does:
+--  - send CreateWindow response with success: true resultCode: "SUCCESS" to app
+-- 3) HMI sends OnSystemCapabilityUpdated(params) notification to SDL
+-- SDL does:
+--  - send OnSystemCapabilityUpdated(params) notification to app
+--  - send OnHMIStatus (hmiLevel, windowID) notification for widget window to app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WidgetSupport/common')
+
+--[[ Local Variables ]]
+local params = {
+  windowID = 1,
+  windowName = "Name of the primary widget",
```

Comment:
Shouldn't this case be a failure? Proposal hints at the primary widget and needing the same name.

From the proposal:

> The primary widget is a special widget, that can be associated with a service type, which is used by the HMI whenever a single widget needs to represent the whole app. The primary widget should be named as the app and can be pre-created by the HMI.

Reply:
@JackLivio It was agreed with proposal author that SDL should not validate `WindowName` for all WIndowIDs = 1 (thus for all PRIMARY WIDGETS)